std/experimental/allocator/building_blocks/kernighan_ritchie.d

```diff
@@ -525,6 +530,15 @@ struct KRRegion(ParentAllocator = NullAllocator)
         return null;
     }
 
+    private void ensureCorrectSize()
+    {
+        import std.math : abs;
+
+        if (root && (abs(payload.length - root.size) < Node.sizeof)) {
```

Comment:
Yes, I was able to reproduce it only when the root is the only node (dealloc has been called on all nodes and we should have an empty allocator). But those leaked bytes exist even if we don't deallocate all nodes and empty is expected to return false. I'm not sure yet if this can be avoided.